cooltools/api/sample.py

```diff
@@ -71,6 +73,10 @@ def sample(
         The target sample size as a fraction of contacts in the original
         dataset. Mutually exclusive with `count`.
 
+    cis_target : bool
```

Comment:
maybe rename to cis_count and move up to be close to the original `count`

Reply:
Done!